rollouts/user-manuals/api-specifications.md

```diff
@@ -104,6 +104,21 @@ There are 3 major parts of api specifications you should pay attention to:
 - Config your deployment strategy before releasing: Tell Rollout how to roll your workload and traffic.
 
 ## API Details
+
+### Rollout Spec Fields
+
+| Field             | Type    | Default | Description                                                              |
+|-------------------|---------|---------|--------------------------------------------------------------------------|
+| `disabled`        | boolean | false   | When true, completely disables Rollout reconciliation                    |
+| `workloadRef`     | Object  |         | Reference to the workload being managed                                  |
+| `strategy`        | Object  |         | Rollout strategy configuration (canary or blue‑green)                     |
+
+**Version Compatibility**
+- `blueGreen` strategy requires Kruise Rollout v0.5.0+
+- `spec.strategy.paused` is available in both v1alpha1 and v1beta1 APIs
```

Comment:
plz also move description of `spec.strategy.paused` here to strategy API section